Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/OldOneClimbyBoi.java

```diff
@@ -0,0 +1,67 @@
+//package org.usfirst.frc.team5493.robot.subsystems;
+//
+//import org.usfirst.frc.team5493.robot.Robot;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)